mqtt-common/src/main/java/org/apache/rocketmq/mqtt/common/util/TopicUtils.java

```diff
@@ -31,7 +31,7 @@ public class TopicUtils {
      * @return
      */
     public static String normalizeTopic(String topic) {
-        if (topic == null) {
+        if (topic == null || topic.isEmpty()) {
```

Comment:
bug of 'empty str'

Reply:
may be use StringUtils.isBlank(topic) will be better

Reply:
> may be use StringUtils.isBlank(topic) will be better

done